### DISCUSSION TITLE
Submissions Next/Previous Button

### DISCUSSION THREAD

**Discussion by kluby** [elementor/elementor#22153] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I'm always frustrated when I'm viewing submissions and I have to either hit the back arrow in my browser or click on Elementor > Submissions in the sidebar again to get back to the list. But even when I do that, I have to reapply filters to make finding my place easier. I currently have over 1500 submissions and going through them in the previously described methods is a huge pain.

### Describe the solution you'd like

While viewing a submission, there should be a next and previous button to view another submission directly. Quick and easy. No roundabout methods.

### Describe alternatives you've considered

If I want to view another submission, at least remember the filters I've already applied, so I don't have to re-enter them to get back to my last spot in the list.

**Comment by UzzolX** [comment]
Absolutely agree, a direct 'Next' and 'Previous' button for submissions would be incredibly convenient.
